data/dashboard.html

```diff
@@ -0,0 +1,560 @@
+<!DOCTYPE html>
+<!--based on https://github.com/infusion/GPS.js/blob/master/examples/dashboard/dahboard.html -->
+<html>
+
+<head>
+        <title>MAIANA GPS Dashboard</title>
+        <meta name="viewport" content="initial-scale=1.0">
+        <meta charset="utf-8">
+
+        <link href="https://fonts.googleapis.com/css?family=Oswald" rel="stylesheet">
```

Comment:
fully agree. 
Need to check if the font file still fits the size of the SPIFF. (I can make the demo/test ais file smaller otherwise)
Alternatively I use the same font you used in in the other pages, then it just needs resizing here &there. (the font came from the GPS example I used)